test/schema/company_test.exs

```diff
@@ -1,12 +1,26 @@
-defmodule Schema.CompanyTest do
+defmodule Companies.Schema.CompanyTest do
   use ExUnit.Case
 
   alias Companies.Schema.Company
 
-  describe "validate_industries/1" do
-    test "returns bool" do
-      assert Company.validate_industries(%Company{industry: "Information Technology"})
-      refute Company.validate_industries(%Company{industry: "Non-existant"})
+  @attrs %{
+    name: "Acme Corp",
+    website: "https://example.com/",
+    github: "",
+    industry: "Information Technology",
+    location: %{"city" => "City", "state" => "State", "country" => "Country"},
+    description: "Description of Acme Corp goes here.",
+    last_changed_on: "2021-01-01"
+  }
```

Comment:
Would it make sense to use either the [alpha.exs](https://github.com/beam-community/elixir-companies/blob/main/test/support/companies/alpha.exs) or [zulu.exs](https://github.com/beam-community/elixir-companies/blob/main/test/support/companies/zulu.exs) factories (with a bit of updating to make them valid)?

Reply:
I thought about it and I want to clean all of that up. Alpha and Zulu exist to be valid and loaded by the nimble publisher for some other tests, they wouldn't work for the invalid companies so need to solve for that.

Reply:
The other reason I didn't use one here is I wanted the parameters to be clear in the test not hidden in a factory or fixtures that's loaded into a variable and passed in.

Reply:
Yeah that makes sense to me. My brain leans towards DRY things but explicit and obvious makes sense. Let me know if I can help with any of the cleanup.